Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -69,16 +73,23 @@ public class BTShopperInsightsClient {
                 isVenmoRecommended: venmo?.recommended ?? false,
                 isEligibleInPayPalNetwork: payPal?.eligibleInPayPalNetwork ?? false || venmo?.eligibleInPayPalNetwork ?? false
             )
-            return self.notifySuccess(with: result)
+            return self.notifySuccess(with: result, for: experiment)
         } catch {
-            throw self.notifyFailure(with: error)
+            throw self.notifyFailure(with: error, for: experiment)
         }
     }
 
     /// Call this method when the PayPal button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
-    public func sendPayPalPresentedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.payPalPresented)
+    /// - Parameters:
+    ///    - paymentMethodsDisplayed: Optional:  The list of available payment methods, rendered in the same order in which they are displayed.
+    ///    - experiment: Optional:  A `JSONObject` passed in as a string containing details of the merchant experiment.
+    public func sendPayPalPresentedEvent(paymentMethodsDisplayed: [String?] = [], experiment: String? = nil) {
```

Comment:
**note**: just checked with product and they suggested to pass in `experiment` here as well since the `experiment` passed into the `presentedEvent` and `getRecommendPaymentMethods` methods can vary

Reply:
Where do the merchants get the values for the `paymentMethodsDisplayed` array?
We don't pass the array in in the demo app.

Reply:
yep "that is correct - this parameter is for the merchant to tell us the order of payment methods displayed. typically, they will pass "PayPal, Venmo or other (for non-PYPL buttons)" mentioned by Arpan Dhandhania